Update readme (#1)
DevExpress-Examples/xaf-how-to-show-a-custom-form-as-the-winforms-dashboard-designer: Readme.md

<!-- default badges list -->
[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T473819)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->

> # XAF - How to show a custom form as the WinForms Dashboard Designer

Reviewer: I have a question about the definite article before WinForms Dashboard Designer. Shouldn't it be "a"?

Reviewer 2: ```suggestion
# XAF - How to show a custom form as a WinForms Dashboard Designer
```

> This example describes how to show a custom form as the WinForms Dashboard Designer used to create and modify dashboards in the WinForms XAF applications. This can be required for the menu customization. For example, add custom BarItems or remove some default BarItems.

Reviewer: ```suggestion
This example describes how to display a custom form as a WinForms Dashboard Designer that is used to create and modify dashboards in WinForms XAF applications. This can be required for menu customization. For example, you may need to add custom BarItems or remove default BarItems.
```
